spec/editors_draft.html

```diff
@@ -3298,6 +3298,232 @@ <h2>Evaluation Semantics</h2>
+      </p>
+        
+      <p>
+        The following sections describe requirements for conforming SPARQL systems that must be applied to composite values during their import into and export from datasets to support proper handling of blank nodes.
+      </p>
+    </section>
+
+    <section>
+      <h2>Importing Requirements</h2>
+      <p>
+        Conforming implementations MUST process <a>cdt:List literal</a>s and <a>cdt:Map literal</a>s during loading, replacing the substrings containing the serialized form of any blank nodes with identifier <var>id</var> with a blank node label <var>b</var>, such that
+      </p>
+      
+      <ul>
+        <li>within the scope of the document being processed, all occurrences of <var>id</var> are replaced with the same <var>b</var> value, and</li>
+        <li>within the scope of the document being processed, two distinct <var>id</var> values are replaced with distinct <var>b</var> values, and</li>
+        <li>
+          if the format of the document being processed has a mechanism for explicitly identifying blank nodes outside of cdt:List and cdt:Map literals (e.g., `_:b1` syntax in N-Triples and in Turtle, `rdf:nodeID` in RDF/XML),
+          and the blank node identifier <var>id</var> appears both inside and outside of such literals,
+          and the parser for this document format emits a blank node <var>B</var> for occurrences of <var>id</var> outside of the literals,
+          then the rewritten blank node label <var>b</var> MUST be a lexical form that maps to <var>B</var> in the lexical-to-value mapping (that is, <i><a>bnl2bn</a></i>(<var>id</var>) = <var>B</var>).
+        </li>
+      </ul>
+
+      <p>
+        Here, "loading" is any single process by which composite values enter a SPARQL system or RDF Dataset. This includes (but is not limited to):
+      </p>
+
+      <ul>
+        <li>The parsing of a SPARQL Query string [[SPARQL11-QUERY]] or of a SPARQL Update string [[SPARQL11-UPDATE]], which may contain literals.</li>
+        <li>The loading of RDF data via a SPARQL <a data-cite="SPARQL11-UPDATE#load">`LOAD`</a> operation [[SPARQL11-UPDATE]]</li>
+        <li>The loading of RDF data via a single SPARQL graph store protocol request [[SPARQL11-HTTP-RDF-UPDATE]]</li>
+        <li>The loading of RDF data via any single SPARQL <a data-cite="SPARQL11-QUERY#unnamedGraph">`FROM`</a> or <a data-cite="SPARQL11-QUERY#namedGraphs">`FROM NAMED`</a> clause [[SPARQL11-QUERY]] in systems that <a data-cite="SPARQL11-SERVICE-DESCRIPTION#sd-dereferencesuris">support the dereferencing and loading of RDF data specified for the dataset to be queried</a></li>
+        <li>Importing of any RDF data to the internally-managed RDF dataset via means other than SPARQL Update statements [[SPARQL11-UPDATE]] or the SPARQL Graph Store HTTP Protocol [[SPARQL11-HTTP-RDF-UPDATE]]</li>
+        <li>The loading of SPARQL Results via [[SPARQL11-FEDERATED-QUERY]]</li>
```

Comment:
Related to https://github.com/awslabs/SPARQL-CDTs/pull/2#discussion_r1561821517

```suggestion
        <li>The loading of a SPARQL result obtained from a SPARQL service when evaluating a <a data-cite="SPARQL11-FEDERATED-QUERY#service">`SERVICE`</a> pattern [[SPARQL11-FEDERATED-QUERY]]</li>
```